src/components/CourseForm/CourseForm.jsx

```diff
@@ -194,7 +229,11 @@ export default function CreateCourse({ history }) {
 						/>
 					</div>
 					<div className='create-course-title-button'>
-						<Button className='app-button' text='Create Course' type='submit' />
+						<Button
```

Comment:
8. [1 point] - Modify course creation functionality:
CreateCourse page and Create course button should be available only for user with the role
ADMIN.